commands.js

```diff
@@ -541,7 +541,7 @@ exports.commands = {
 		"/publicroom - Makes a room public. Requires: \u2605 & ~"],
 
 	modjoin: function (target, room, user) {
-		if (!target) return this.parse('/help modjoin');
+		if (!target) return this.sendReply("Modjoin is currently set to: " + (room.modjoin ? room.modjoin : false));
```

Comment:
I want to specify here that the reason why I put this statement was that some rooms that were created before the rooms.js changes on this PR could potentially have their modjoin level set to ``undefined``, and I think displaying ``false`` like modjoin does looks better.